pkgs/servers/http/apache-modules/mod_python/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
 
   passthru = { inherit apacheHttpd; };
 
-  buildInputs = [ apacheHttpd python2 ];
+  buildInputs = [ apacheHttpd python2 libintl ];
```

Comment:
```suggestion
  buildInputs = [ apacheHttpd python2 ] ++ lib.otpional stdenv.isDarwin libintl;
```

Reply:
libintl is defined as null for glibc platforms: https://github.com/NixOS/nixpkgs/blob/b1eb90c00fedd4f6b15763e1ea6fafaa4338543a/pkgs/top-level/all-packages.nix#L16051-L16052

Reply:
That should never be the case.

Reply:
I don't mind, but also feel like someone will end up having to rethink all of this if, e.g., *bsd support becomes serious. 🙂